content/client-server-api/modules/voip_events.md

```diff
@@ -174,16 +174,34 @@ In response to an incoming invite, a client may do one of several things:
 Clients may send more than one stream in a VoIP call. The streams should be
 differentiated by including metadata in the [`m.call.invite`](/client-server-api/#mcallinvite),
 [`m.call.answer`](/client-server-api/#mcallanswer) and [`m.call.negotiate`](/client-server-api/#mcallnegotiate)
-events, using the `sdp_stream_metadata` property.
+events, using the `sdp_stream_metadata` property. An [`m.call.sdp_stream_metadata_changed`](/client-server-api/#mcallsdp_stream_metadata_changed)
+event can be sent when the metadata changes but no negotiation is required.
 
 `sdp_stream_metadata` maps from the `id` of a stream in the session description, 
-to metadata about that stream. Currently only one property is defined for the
-metadata. This is `purpose`, which should be a string indicating the purpose of
-the stream. The following `purpose`s are defined:
+to metadata about that stream. Currently three properties are defined for the
+metadata: `purpose`, `audio_muted` and `video_muted`.
+
+`purpose` should be a string indicating the purpose of the stream. The following
+`purpose`s are defined:
 
 *  `m.usermedia` - stream that contains the webcam and/or microphone tracks
 *  `m.screenshare` - stream with the screen-sharing tracks
 
+`audio_muted` and `video_muted` are booleans that indicate whether the audio and
+video tracks in the stream are muted, respectively. All tracks should be assumed
+unmuted unless specified otherwise.
```

Comment:
Given that this stuff is repeated in the definition of each event, it feels a bit  redundant to repeat it here. Let's just rip it out?

Reply:
The purpose stuff too?

Reply:
I also removed line 180 since it's also in the description of the events.